### PR TITLE
[BEAM-3735] Copy mobile-gaming sources in to archetypes

### PR DESCRIPTION
**Pull request by yifanzou**
DESCRIPTION HERE

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by yifanzou** [comment]
+R: @alanmyrvold

**Comment by yifanzou** [comment]
@lukecwik I talked to Huygaa and modified the code. It will only copy game examples including injector and utils instead of all stuffs from */complete/. Please review when you get a moment. Thanks~

File: sdks/java/maven-archetypes/examples/generate-sources.sh [hunk 1]
```diff
@@ -45,6 +45,20 @@ rsync -a --exclude cookbook --exclude complete --exclude '*IT.java'  \
     "${ARCHETYPE_ROOT}/src/test/java"				                 \
```
Comment (lukecwik) [inline sdks/java/maven-archetypes/examples/generate-sources.sh]: For both rsync lines above, we should exclude the snippets directory from being copied over from the examples to the archetypes.

Comment (yifanzou) [reply sdks/java/maven-archetypes/examples/generate-sources.sh]: done.

**Comment by robertwb** [comment]
@lukecwik Are you happy with this now?

**Pull request merged**